hub/powertoys/always-on-top.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: PowerToys Always on Top utility for Windows
+title: PowerToys Always On Top utility for Windows
```

Comment:
It's not correct according to the style guide, but the capitalized "On" matches the name of the utility in PowerToys.